src/SignUpIn/SignIn.js

```diff
@@ -34,63 +63,97 @@ class SignIn extends Component {
-                                type="email"
-                                id="email"
-                                className="login-input form-component"
-                                placeholder="Email"
-                                required
-                            />
-                            <br />
-                            <input
-                                name="password"
-                                type="password"
-                                id="password"
-                                className="login-input form-component"
-                                placeholder="Password"
-                                required
-                            />
-                            <br />
-                        </div>
-                        {this.props.signInDenied && (
-                            <p className="sign-in-error">
-                                Log in denied. Account is not verified or
-                                activated.
-                            </p>
-                        )}
-                        {this.state.error && (
-                            /* TODO: give better error msg */
-                            <p className="sign-in-error">
-                                Unable to log in.
-                            </p>
-                        )}
-                        <div className="login-button-wrapper">
-                            <button type="submit" className="login-button">
-                                login
-                            </button>
-                        </div>
-                        <div className="forgot">
-                            {/* TODO: Add functionality to reset username and password */}
-                            {/* <p className="forgot">forgot password?</p> */}
-                            {/* <p className="forgot">forgot username?</p> */}
-                        </div>
-                        <div className="signup-reroute">
-                            {/* TODO: Link straight to signup when login routing is ready */}
-                            {/* <Link to={'/signup'}>Create Account</Link> */}
-                            <div className="back" onClick={this.props.back}>
-                                Back
+                            <div className="login-input-wrapper">
+                                <input
+                                    name="email"
+                                    type="email"
+                                    id="email"
+                                    className="login-input form-component"
+                                    placeholder="Email"
+                                    required
+                                />
+                            </div>
+                            <div className="login-button-wrapper">
+                                <button type="submit" className="login-button">
+                                     Send Password
```

Comment:
Nitpicky thing but maybe saying like "Reset Password" would be a bit better than "Send Password"? "Send Password" makes it seem like we're just gonna email them their current password